ls1intum/Artemis
# `Programming exercises`: Add information to feedback view

### Checklist
#### General


#### Client

### Motivation and Context


Students should know how many feedbacks they have in each category.

### Description
- Adds the amount of feedbacks per group as a number after the name `Wrong -> Wrong (3)`
- For feedbacks: Adds a gap between the amount of points and the feedback title 
- Adds a tooltip on hover for the date in the submission information (see screenshot below)

### Steps for Testing

Prerequisites:
Go to a feedback detail view and check if there are any issues and if it looks like in the screenshots

### Review Progress






### Test Coverage





### Screenshots



- Amount of feedback next to category name (Dont mind the blue background. I accidentally selected the points)
![Screenshot_20230109_163208](https://user-images.githubusercontent.com/25489463/211345712-37f8109b-4c79-4f9f-a1b9-53d7ef15f9df.png)

- The purple dotted line between title and amount of points is the minimal padding
![Screenshot_20230110_123730](https://user-images.githubusercontent.com/25489463/211542194-4735d260-d5c1-4d0b-a16c-24305270c69d.png)

- Date tooltip on hover
![Screenshot_20230110_125439](https://user-images.githubusercontent.com/25489463/211545293-1d36d1c1-8815-4daf-9784-03a84585c9b4.png)
